workspaces/applications/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+cargo-features = ["resolver"]
+
+[workspace]
+members = [
+  # "crates/sdk/rust-examples/audio-event-triangulation",
```

Comment:
Why is this commented out?

Reply:
It does not build:
```error[E0599]: no method named `checked_div` found for type `f64` in the current scope
  --> crates/sdk/rust-examples/audio-event-triangulation/src/main.rs:49:11
   |
49 |         p.checked_div(self.samples.len() as f64).unwrap_or(0)
   |           ^^^^^^^^^^^ method not found in `f64`
```

It is not built by the current build system. I don't know if it should be removed or fixed.